_faq/is-mentor-necessary.md

```diff
@@ -1,6 +1,5 @@
 ---
-question: "A showcased project on the website doesn't have a mentor, but
-	I'm interested in that project?"
+question: "A showcased project on the website doesn't have a mentor, but I'm interested in that project?"
```

Comment:
no , please do as requested in the last review.

(your change makes it go over the preferred line length)